.contributing/the-process.md

```diff
@@ -1,120 +1,128 @@
-0 - Introduction 
-=================
+# 0 - Introduction 
 
 Note that the sections are numbered but the items within each section are dot points rather than a numbered list. The two sections should be done in order, but within each section it's probably a good idea to jump around and expect to do multiple passes.
 
-1 - Planning in consultation with stakeholders 
-===============================================
+# 1 - Planning in consultation with stakeholders 
 
--   Replace *Section* below with the section name, eg. Auth
+- Replace _Section_ below with the section name, eg. Auth
 
--   Use the next available Section number (denoted x. below)
+- Use the next available Section number (denoted x. below)
 
--   Create in Drive - a folder called "x. *Section*: Sections Planning" in "Sections Planning (CL)".  In the folder: 
+- Create in Drive - a folder called "x. _Section_: Sections Planning" in "Sections Planning (CL)".  In the folder:
 
-    -   a Google doc called "0 - Use Cases < *Section* (CL)" that will hold just a list of Use Cases, so we have a short, easy to read and think about, list that is the definitive set of use cases
+  - a Google doc called "0 - Use Cases < _Section_ (CL)" that will hold just a list of Use Cases, so we have a short, easy to read and think about, list that is the definitive set of use cases
 
-        -   short, clear names → balance brevity with clarity 
+    - short, clear names → balance brevity with clarity
 
--   Create an Asana Project called "x. *Section*" 
+- Create an Asana Project called "x. _Section_"
```

Comment:
ahhh this is vscode auto stuff, ill fix it!

Reply:
Oh cool, auto formatting doesn't happen for me... maybe I should change my settings - did you add a setting to get autoformatting for md files? 

The weird formatting came from https://euangoddard.github.io/clipboard2markdown/ which I used to convert the google doc to a markdown file.  

Personally I'd be happy to go with the VS Code formatting.  It'd be good to do it in a separate PR I reckon.